Frends.GooglePubSub.Publish/Frends.GooglePubSub.Publish/Definitions/Message.cs

```diff
@@ -0,0 +1,76 @@
+﻿using Google.Cloud.PubSub.V1;
+using Google.Protobuf;
+using System.Linq;
+
+namespace Frends.GooglePubSub.Publish.Definitions;
+
+/// <summary>
+/// Message attribute.
+/// </summary>
+public class MessageAttribute
+{
+    /// <summary>
+    /// Default ctor.
+    /// </summary>
+    public MessageAttribute()
+    { }
+
+    /// <summary>
+    /// Ctor with key and value params.
+    /// </summary>
+    /// <param name="key"></param>
+    /// <param name="value"></param>
+    public MessageAttribute(string key, string value)
```

Comment:
This constructor is only used in unit tests, to it can be removed.
In unit tests you can initiate this class with `new MessageAttribute{Key = "key", Value = "value"}`.

Reply:
Is there any particular reason why you consider this as something bad?

Reply:
Well not necessarily bad. It's just extra code, since all parameters have public getters/setters, so it is possible to create instance of the class without providing constructor. In addition, now you have to define the empty constructor also, so it's again couple of lines more. This is also deviance from other Tasks.

Reply:
So, personal coding preferences aside - is this something that you require to be changed before the PR could be merged?